deploy/crds/litmuschaos_v1alpha1_chaostemplate_cr.yaml

```diff
@@ -1,7 +1,36 @@
-apiVersion: litmuschaos.io/v1alpha1
+---
+
+## The resource that contains the actual/low-level chaos params.
+## Consists of executor definition in its spec, with ENV default
+## values and entrypoint scripts (playbooks). The chaosGraph is
+## referenced in the chaostype parameter of the chaosExperiment
+## resource. Consists of default values for chaos-specific params
+## which are expected to be overridden from the chaosExperiment  
+
+apiVersion: litmus.io/v1alpha1
 kind: ChaosTemplate
-metadata:
-  name: example-chaostemplate
+metadata: 
+  name: simple-pod-failure
+  namespace: 
 spec:
-  # Add fields here
-  size: 3
+  definition:
+    labels: 
+      name: simple-pod-failure
```

Comment:
IMO name has to be very specific.
Pod failure can be done via various ways, e.g.:
- pod delete
- pod oom
- pod's image not found
- etc.

Reply:
Ok. will update